docker-compose.dev.yml

```diff
@@ -19,26 +19,33 @@ services:
   validator:
     image: hyperledger/sawtooth-validator:1.0.1
     depends_on:
-      - remme-genesis
       - settings-tp
+    volumes:
+      - validator_keys:/etc/sawtooth/keys
     expose:
       - 4004
-    volumes:
-      - user_keys:/root/.sawtooth/keys
-      - genesis_data:/genesis/batch
+    ports:
+      - '${REMME_VALIDATOR_PORT:-8800}:8800'
+    env_file:
+      - .env
     entrypoint: |
       bash -c "
-        sawadm keygen && \
-        sawtooth keygen key
-        if [ -d "/var/lib/sawtooth" ]; then
-          rm /var/lib/sawtooth/*
+        if [ ! -f /etc/sawtooth/keys/validator.priv ]; then
+          sawadm keygen
+        fi
+        ADDITIONAL_ARGS=\"--peering $$REMME_PEERING_MODE\"
+        if [ ! -z \"$$REMME_PEERS_LIST\" ]; then
+          ADDITIONAL_ARGS=\"$$ADDITIONAL_ARGS --seeds $$REMME_PEERS_LIST\"
+        fi
+        if [ ! -z \"$$REMME_SEEDS_LIST\" ]; then
+          ADDITIONAL_ARGS=\"$$ADDITIONAL_ARGS --seeds $$REMME_SEEDS_LIST\"
         fi
-        sawset genesis -k /root/.sawtooth/keys/key.priv && \
-        sawadm genesis config-genesis.batch /genesis/batch/token-proposal.batch && \
         sawtooth-validator -vv \
-          --endpoint tcp://validator:8800 \
+          --endpoint tcp://$$REMME_VALIDATOR_IP:$$REMME_VALIDATOR_PORT \
           --bind component:tcp://eth0:4004 \
-          --bind network:tcp://eth0:8800
+          --bind network:tcp://eth0:8800 \
```

Comment:
"8800 internal port is static" 
Do we not consider the case of VPS running several validators and TPs at once? 
Will there be an internal `port` conflict with other "instances"?
Or 1 VPS = 1 Validator?

Reply:
1 VPS = 1 Validator I think. If that changes we will adapt, not a big piece of work.